osu.Game/Screens/Play/DimmableStoryboard.cs

```diff
@@ -19,6 +20,14 @@ public class DimmableStoryboard : UserDimContainer
         private readonly Storyboard storyboard;
         private DrawableStoryboard drawableStoryboard;
 
+        /// <summary>
+        /// Whether the storyboard is considered finished.
+        /// </summary>
+        /// <remarks>
+        /// This is true by default in here, until an actual drawable storyboard is loaded, in which case it'll bind to it.
+        /// </remarks>
+        public IBindable<bool> HasStoryboardEnded = new BindableBool(true);
```

Comment:
To reiterate on https://github.com/ppy/osu/pull/12432#discussion_r615210904, this is how I thought about having it https://github.com/frenzibyte/osu/commit/425d6b42ffcf52c2faee8fc0167d1ae73d011df1, while the current solution pushed here is simpler, the changes I made in the linked commit keeps `DimmableStoryboard` from having to expose things in `DrawableStoryboard` and initialize with a sane default.

I'm fine either way, but going to leave this as a comment.

Reply:
@frenzibyte your version is a bit weird (private event for some reason with a separate binding method, instead of just having the event public and then invoking it at the right time), and I honestly prefer the variant as written here. Makes sense to me to have `HasStoryboardEnded` on both `DrawableStoryboard` and this.

Reply:
> @frenzibyte your version is a bit weird (private event for some reason with a separate binding method, instead of just having the event public and then invoking it at the right time)

actually yeah could make it public and assign the event at `createUnderlayComponents` right after creating the dimmable before returning it for load, didn't think about it.